notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<p>Let's have a nice metric definition like you do in other sections :-) </p><p>Context Utilization is.... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>